modules/dr-restoring-etcd-quorum-ha.adoc

```diff
@@ -13,29 +13,43 @@ You can use the `quorum-restore.sh` script to instantly bring back a new single-
 You might experience data loss if the host that runs the restoration does not have all data replicated to it.
 ====
 
+[IMPORTANT]
+====
+Quorum restoration should not be used to decrease the number of nodes outside of the restoration process. Decreasing the number of nodes results in an unsupported cluster configuration.
+====
+
 .Prerequisites
 
 * You have SSH access to the node used to restore quorum.
 
 .Procedure
 
-. Select a control plane host to use as the recovery host. You run the restore operation on this host.
+. Select a control plane host to use as the recovery host. You run the restore operation on this host. You can choose any of the surviving nodes to run the recovery on.
 
 . Using SSH,  connect to the chosen recovery node and run the following command to restore etcd quorum:
 +
 [source,terminal]
 ----
 $ sudo -E /usr/local/bin/quorum-restore.sh
 ----
++
+After a few minutes, the nodes that went down are automatically synchronized with the node that the recovery script was run on. Any remaining online nodes automatically rejoin the new etcd cluster created by the `quorum-restore.sh` script. This process takes a few minutes.
 
 . Exit the SSH session.
 
+. Return to a three-node configuration if any nodes are offline. Use the original installation method, installer-provisioned or user-provisioned, to replace the nodes. Nodes automatically join the etcd cluster once reprovisioned. This process takes some time.
+
```

Comment:
A node can be lost due to network split (which is fixed by recovering network access to the standing cluster node(s) or hardware failure, which might certainly require reinstallation. There might be other causes too. 
I don't know if the recovery/reinstallation can be done using the openshift-installer as explained (IPI/UPI methods) though. Can you confirm this?

Reply:
In IPI or UPI with Machine API, all that should need to be done to replace a node is to delete the bad machine(s) and they'll be re-added by the Machine API and then the CEO will reconcile the etcd instances.
In UPI without machine api, then the user has to recreate the machine in whatever way they originally created. 

Could probably pull these steps and language from the old restoration docs.

Reply:
Does this depend on ControlPlaneMachineSet being supported? Bare metal is the main target of this feature and support for it isn't formalized https://docs.openshift.com/container-platform/4.17/machine_management/control_plane_machine_management/cpmso-getting-started.html#cpmso-platform-matrix_cpmso-getting-started if this is a requirement, that is.

Reply:
I'm referencing the old docs for disaster recovery where machines are needed to be replaced after restore: step 15 https://docs.openshift.com/container-platform/4.17/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

If a node is alive, then everything should just reconcile, but if there's a node that's offline (or the datacenter burnt down), then the user will need to replace the node; I was thinking of just reusing this language since it's worked in the past.

Reply:
Thanks for pointing to the source of the info, that makes sense.

Reply:
> Could probably pull these steps and language from the old restoration docs

Agreed, or even point to them? Wdyt @kowen-rh?

Reply:
We can pull the steps from the previous documentation IMO.